src/pages/blog/cluster-api-development-environment.md

```diff
@@ -0,0 +1,197 @@
+---
+templateKey: blog-post
+title: 'Setting up a development environment with Cluster API'
+author: Alexander Hughes
+date: 2020-04-09T12:00:00.000Z
+category: 
+  - label: Airship 2.0
+    id: category-C98iZYrE1
+---
+
+Airship is a collection of loosely coupled, but interoperable open source tools that declaratively automates cloud
+provisioning. Airship is designed to make your cloud deployments simple, repeatable, and resilient.
+
+The primary motivation for Airship 2.0 is the continued evolution of the control plane, and by aligning with maturing
+CNCF projects we can improve Airship by making 2.0:
+- More capable
+- More secure
+- More resilient
+- Easier to operate
+
+<br>
+
+One such project is [Cluster API](https://cluster-api.sigs.k8s.io/), a Kubernetes project that brings declarative,
+Kubernetes-style APIs to cluster creation, configuration, and management. It provides optional, additive functionality
+on top of core Kubernetes to manage the lifecycle of a Kubernetes cluster.
+
+In a previous [blog post](
+https://www.airshipit.org/blog/airship-blog-series-5-drydock-and-its-relationship-to-cluster-api/), Alan Meadows and
+Rodolfo Pacheco discussed the evolution of Airship 1.0 to Airship 2.0 and the relationship between [Drydock](
+https://opendev.org/airship/drydock) and [Cluster API](https://cluster-api.sigs.k8s.io/). It's an interesting read,
+looking at how Cluster API will be used by Airship 2.0.
+
+Today I will provide you the documentation and my tested step-by-step directions to creating a Cluster API development
```

Comment:
"to creating"

Reply:
not clear on your comment, is this for line 33?  Current wording is "Today I will provide you the documentation and my tested step-by-step directions to creating a Cluster API development"